src/util/mock-servers/service-testers/stream-tester.ts

```diff
@@ -0,0 +1,59 @@
+// Copyright 2022 Google LLC
```

Comment:
I would have anything related to testing in the `test` folder rather than `src` (which should exclusively be application logic).